packages/icons/icons.json

```diff
@@ -4198,5 +4198,12 @@
         "tags": "data, type, format，punctuation, grammar",
         "group": "miscellaneous",
         "codepoint": 62296
+    },
+    {
+        "displayName": "Clipboard: file",
+        "iconName": "clipboard-file",
+        "tags": "paste, attach, file",
+        "group": "action",
+        "codepoint": 62297
```

Comment:
can you merge in develop? we need to increment this now (sorry) https://github.com/palantir/blueprint/pull/6731/files#diff-85aa933a7e3200e367137ef06dfbac6ad751ca2f91ed0bb8c8e8f61e20b925b6R4214